src/lib/types/evm.ts

```diff
@@ -19,6 +24,21 @@ export enum EvmMethodName {
   CallErc20Factory = "Call ERC20 Factory",
 }
 
+export type EvmToAddress =
+  | {
+      toType: EvmMethodName.Create;
```

Comment:
Could we use `methodType` instead of `toType`?

Reply:
we didn't handle all method types so I prefer to not to use `method`